builds/opendingux/gcw-zero/readme.txt

```diff
@@ -4,8 +4,8 @@ Installation Instructions
 RTP
 ---
 * Install the RTP files on your computer
-* RPG2K games: Copy the RTP files, not the RTP folder iteself, to /media/data/local/share/easyrpg/rtp2k
-* RPG2K3 games: Copy the RTP files, not the RTP folder itself, to /media/data/local/share/easyrpg/rtp2k3
+* RPG2K games: Copy the RTP files, not the RTP folder iteself, to /usr/local/home/.easyrpg/rtp2k
+* RPG2K3 games: Copy the RTP files, not the RTP folder itself, to /usr/local/home/.easyrpg/rtp2k3
 
```

Comment:
Maybe we should simply mention "home directory" here. Opinions?

Reply:
If this one is the only valid folder (or at least an always valid one), it prevents unnecessary user questions because they didn't understand the documentation properly IMHO.

Reply:
@fdelapena: I suggest looking here, it might answer your question: https://github.com/EasyRPG/Player/blob/master/builds/opendingux/gcw-zero/run.sh#L16-L19

Reply:
Most GCW0 users are not very fluent with Linux and don't know what i'm talking about when i say Home folder.
I think it's just less trouble to just post the full path instead.